docs/HowTo/Develop-Dapps/Use-web3js.md

```diff
@@ -1,59 +1,72 @@
-description: Hyperledger Besu client libraries 
-<!--- END of page meta data -->
+---
+description: Hyperledger Besu client libraries
+---
 
-# Create a Signed Transaction using Web3.js
+# Create a signed transaction using Web3.js
 
-You can use the example Javascript scripts to create and send raw transactions in the private network 
-created by the [Private Network Example](../../Tutorials/Examples/Private-Network-Example.md).
+You can use the example JavaScript scripts to create and send raw transactions in the private
+network created by the
+[Private Network Example](../../Tutorials/Examples/Private-Network-Example.md).
 
-Update the `JSON-RPC endpoint` in the following examples to the endpoint for the private 
-network displayed after running the `./run.sh` script.
+In the following examples update the `JSON-RPC endpoint` to the endpoint for the private network
+displayed after running the `./run.sh` script.
 
 !!! warning "Private keys"
 
     Do not use the accounts in the examples on mainnet or any public network except for testing.
-    The private keys are displayed which means the accounts are not secure.
+    They display the private keys, which means the accounts are not secure.
     
-    All accounts and private keys in the examples are from the `dev.json` genesis file in the 
-    [`/besu/ethereum/core/src/main/resources`](https://github.com/hyperledger/besu/tree/master/config/src/main/resources) directory.
-
-    In production environments avoid exposing your private keys by creating signed transactions 
-    offline, or use [EthSigner](https://docs.ethsigner.pegasys.tech/) to isolate your private keys and 
-    sign transactions with [`eth_sendTransaction`](https://docs.ethsigner.pegasys.tech/Using-EthSigner/Using-EthSigner/#eth_sendtransaction). 
+    All accounts and private keys in the examples are from the `dev.json` genesis file in the
+    [`/besu/ethereum/core/src/main/resources`](https://github.com/hyperledger/besu/tree/master/config/src/main/resources)
+    directory.
+
+    In production environments avoid exposing your private keys by creating signed transactions
+    offline, or use [EthSigner](https://docs.ethsigner.pegasys.tech/) to isolate your private keys
+    and sign transactions with
+    [`eth_sendTransaction`](https://docs.ethsigner.pegasys.tech/Using-EthSigner/Using-EthSigner/#eth_sendtransaction).
     
```

Comment:
Same here for `./docs/HowTo/Develop-Dapps/Use-web3js.md:26: MD009 Trailing spaces` where the spaces are on line 27.
You can configure your IDE to trim spaces even on blank lines automatically or use a regex to search and replace trailing spaces: `[ ]+$` will well work to detect all extra spaces at the end of a line.

It will also fix `./docs/HowTo/Develop-Dapps/Use-web3js.md:27: MD022 Headers should be surrounded by blank lines` as the spaces are not considered as a blank line.

Reply:
Fixed.